front/app/containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm/index.tsx

```diff
@@ -182,45 +186,49 @@ const CustomPageSettingsForm = ({
-                  <span>{formatMessage(messages.linkedProjectsLabel)}</span>
-                  <IconTooltip
-                    ml="10px"
-                    content={formatMessage(messages.linkedProjectsTooltip)}
-                  />
-                </Label>
-              </Box>
-              <Box mb="30px">
-                <Tabs name="projects_filter_type" items={projectsFilterTabs} />
-              </Box>
-              {methods.watch('projects_filter_type') === 'topics' && (
-                <Box mb="30px">
-                  <MultipleSelect
-                    name="topic_ids"
-                    options={mapFilterEntityToOptions(topics)}
-                    label={
-                      <>
-                        {formatMessage(messages.selectedTagsLabel)}
-                        <IconTooltip content={'add some tags'} />
-                      </>
-                    }
-                  />
+            {advancedCustomPagesEnabled && (
+              <Box>
+                <Box display="flex" justifyContent="flex-start">
+                  <Label>
+                    <span>{formatMessage(messages.linkedProjectsLabel)}</span>
+                    <IconTooltip
+                      ml="10px"
+                      content={formatMessage(messages.linkedProjectsTooltip)}
+                    />
+                  </Label>
                 </Box>
-              )}
-              {methods.watch('projects_filter_type') === 'areas' && (
-                <Box mb="20px">
-                  <Select
-                    name="area_id"
-                    options={mapFilterEntityToOptions(areas)}
-                    label={<>{formatMessage(messages.selectedAreasLabel)}</>}
-                    labelTooltipText="choose an area"
+                <Box mb="30px">
+                  <Tabs
+                    name="projects_filter_type"
+                    items={projectsFilterTabs}
                   />
                 </Box>
-              )}
-            </Box>
+                {methods.watch('projects_filter_type') === 'topics' && (
+                  <Box mb="30px">
+                    <MultipleSelect
+                      name="topic_ids"
+                      options={mapFilterEntityToOptions(topics)}
+                      label={
+                        <>
+                          {formatMessage(messages.selectedTagsLabel)}
+                          <IconTooltip content={'add some tags'} />
+                        </>
+                      }
+                    />
+                  </Box>
+                )}
+                {methods.watch('projects_filter_type') === 'areas' && (
+                  <Box mb="20px">
+                    <Select
+                      name="area_id"
+                      options={mapFilterEntityToOptions(areas)}
+                      label={<>{formatMessage(messages.selectedAreasLabel)}</>}
+                      labelTooltipText="choose an area"
+                    />
```

Comment:
You can pass a string without wrapping with `<></>`

Reply:
Good point. Done.